src/frontend/components/DashboardVideoLiveControlPane/tab/DashboardVideoSessionLiveTabAttendance/index.tsx

```diff
@@ -0,0 +1,155 @@
+    id: 'components.DashboardVideoLiveControlPane.tab.DashboardVideoLivettendance.connectedPartially',
+  },
+  dotWasPresent: {
+    defaultMessage: 'Present',
+    description: 'Dot icon showing this slice of time the user was connected',
+    id: 'components.DashboardVideoLiveControlPane.tab.DashboardVideoLivettendance.dotWasPresent',
+  },
+  dotMissed: {
+    defaultMessage: 'Missed',
+    description:
+      'Title on dot icon showing this slice of time the user was not connected',
+    id: 'components.DashboardVideoLiveControlPane.tab.DashboardVideoLivettendance.dotMissed',
+  },
+  notConnected: {
+    defaultMessage: 'Missed the live',
+    description: 'Title on icon to indicate that the user was not present',
+    id: 'components.DashboardVideoLiveControlPane.tab.DashboardVideoLivettendance.notConnected',
+  },
+});
+
+interface DashboardVideoSessionLiveTabAttendanceProps {
+  liveSession: LiveAttendance;
+}
+export const DashboardVideoSessionLiveTabAttendance = ({
+  liveSession,
+}: DashboardVideoSessionLiveTabAttendanceProps) => {
+  // over 80% of attendance, the user is considered very diligent
+  const diligentLimit = 80;
+  const isConnected = (liveAttendanceInfo: LiveAttendanceInfo) => {
+    return (
+      liveAttendanceInfo.onStage ||
+      liveAttendanceInfo.connectedInBetween ||
+      liveAttendanceInfo.playing
+    );
+  };
+
+  const getFormattedInfo = (session: LiveAttendance) => {
+    let nbConnected = 0;
+    const length = Object.keys(session.live_attendance!).length;
+
+    const info = Object.values(session.live_attendance!).map(
+      (liveAttendance) => {
+        const connected = isConnected(liveAttendance as LiveAttendanceInfo)!;
+        if (connected) nbConnected++;
+        return connected;
+      },
+    );
+
+    return {
+      percent: length ? Math.round((nbConnected / length) * 100) : 0,
+      info,
+    };
+  };
+
+  const infoSession = getFormattedInfo(liveSession);
+  const zeroPercent = infoSession.percent === 0;
+  const intl = useIntl();
+  return (
+    <Box
+      align="center"
+      justify="between"
+      direction="row"
+      gap="small"
+      pad={{ horizontal: 'medium', vertical: 'xsmall' }}
+      data-testid="live-attendance"
+    >
+      <Box direction="row" pad="medium">
+        {infoSession.percent >= diligentLimit ? (
+          <StatusGood
+            a11yTitle={intl.formatMessage(messages.connectedDiligent)}
+            color="blue-active"
+            key={liveSession.id}
+          />
+        ) : zeroPercent ? (
+          <Clear
+            a11yTitle={intl.formatMessage(messages.notConnected)}
+            color="red-active"
+            key={liveSession.id}
+          />
+        ) : (
+          <CircleAlert
+            a11yTitle={intl.formatMessage(messages.connectedPartially)}
+            color="blue-active"
+            key={liveSession.id}
+          />
+        )}
```

Comment:
Maybe compute this in the render function outside the jsx

Reply:
done